mobile-app/app/screens/AppNavigator/screens/Loans/components/LoanCardsV2.tsx

```diff
@@ -0,0 +1,202 @@
+  loanTokenId: string;
+  symbol: string;
+  displaySymbol: string;
+  price?: ActivePrice;
+  interestRate: string;
+  onPress: () => void;
+  testID: string;
+  isBorrowHidden: boolean;
+}
+
+export function LoanCardsV2(props: LoanCardsProps): JSX.Element {
+  const ref = useRef(null);
+  useScrollToTop(ref);
+  const navigation = useNavigation<NavigationProp<LoanParamList>>();
+  const vaults = useSelector((state: RootState) => vaultsSelector(state.loans));
+  const activeVault = vaults.find(
+    (v) =>
+      v.vaultId === props.vaultId && v.state !== LoanVaultState.IN_LIQUIDATION
+  ) as LoanVaultActive;
+  return (
+    <>
+      <ThemedFlashList
+        estimatedItemSize={2}
+        contentContainerStyle={tailwind("pt-4 pb-2")}
+        data={props.loans}
+        ref={ref}
+        numColumns={2}
+        renderItem={({
+          item,
+          index,
+        }: {
+          item: LoanToken;
+          index: number;
+        }): JSX.Element => {
+          return (
+            <View style={{ flexBasis: "100%" }}>
+              <LoanCard
+                symbol={item.token.symbol}
+                displaySymbol={item.token.displaySymbol}
+                interestRate={item.interest}
+                price={item.activePrice}
+                loanTokenId={item.tokenId}
+                onPress={() => {
+                  navigation.navigate({
+                    name: "BorrowLoanTokenScreen",
+                    params: {
+                      loanToken: item,
+                      vault: activeVault,
+                    },
+                    merge: true,
+                  });
+                }}
+                testID={`loan_card_${index}`}
+                isBorrowHidden={!props.vaultExist}
+              />
+            </View>
+          );
+        }}
+        keyExtractor={(_item, index) => index.toString()}
+        testID={props.testID}
+      />
+    </>
+  );
+}
+
+function LoanCard({
+  symbol,
+  displaySymbol,
+  price,
+  interestRate,
+  onPress,
+  testID,
+  isBorrowHidden,
+}: LoanCardOptions): JSX.Element {
+  const LoanIcon = getNativeIcon(displaySymbol);
+  const currentPrice = getPrecisedTokenValue(getActivePrice(symbol, price));
+  return (
+    <ThemedViewV2
+      testID={`loan_card_${displaySymbol}`}
+      light={tailwind("bg-mono-light-v2-00")}
+      dark={tailwind("bg-mono-dark-v2-00")}
+      style={tailwind("p-4 mx-2 mb-4 rounded-lg-v2")}
+    >
+      <View style={tailwind("flex-row items-center pb-2 justify-between")}>
+        <View
+          style={tailwind("flex flex-row justify-between items-center w-full")}
+        >
+          <ThemedTextV2
+            light={tailwind("text-mono-light-v2-700")}
+            dark={tailwind("text-mono-dark-v2-700")}
+            style={tailwind("font-medium font-normal-v2")}
+            testID={`${testID}_display_symbol`}
+          >
+            {displaySymbol}
+          </ThemedTextV2>
+          <LoanIcon width={36} height={36} />
```

Comment:
<img width="509" alt="image" src="https://user-images.githubusercontent.com/13292662/192182227-3cefd7b1-d545-4267-b63d-fff4f5507124.png">
The text inside the icon is not using the correct font